modular_skyrat/modules/sec_haul/code/security_medic/secmed_clothes.dm

```diff
@@ -36,20 +36,21 @@
 	supports_variations_flags = CLOTHING_DIGITIGRADE_VARIATION_NO_NEW_ICON
 	allowed = list(/obj/item/analyzer, /obj/item/stack/medical, /obj/item/dnainjector, /obj/item/reagent_containers/dropper, /obj/item/reagent_containers/syringe, /obj/item/reagent_containers/hypospray, /obj/item/healthanalyzer, /obj/item/flashlight, /obj/item/reagent_containers/cup/bottle, /obj/item/reagent_containers/cup/beaker, /obj/item/reagent_containers/pill, /obj/item/storage/pill_bottle, /obj/item/paper, /obj/item/melee/baton/telescopic, /obj/item/soap, /obj/item/tank/internals/emergency_oxygen, /obj/item/gun, /obj/item/storage/medkit)
 
-/obj/item/clothing/under/rank/security/peacekeeper/security_medic
+/obj/item/clothing/under/rank/security/peacekeeper/security_medic/alternate
 	name = "security medics's uniform"
 	desc = "A lightly armored uniform worn by Nanotrasen's Asset Protection Medical Corps."
 	icon = 'modular_skyrat/master_files/icons/obj/clothing/under/security.dmi'
 	worn_icon = 'modular_skyrat/master_files/icons/mob/clothing/under/security.dmi'
 	worn_icon_digi = 'modular_skyrat/master_files/icons/mob/clothing/uniform_digi.dmi'
 	icon_state = "security_medic_jumpsuit"
 
-/obj/item/clothing/under/rank/security/peacekeeper/security_medic/alternate
+/obj/item/clothing/under/rank/security/peacekeeper/security_medic
 	name = "security medics's turtleneck"
```

Comment:
I don't think Medics's is right. Do we even use possessive 's for most other uniforms in the game? Could this just be `security medic [gear]`?
Either choice tho s's isn't right and a few items in here use that, may as well fix it while you're here